common/configuration/class.FileSystemComponent.php

```diff
@@ -244,38 +244,54 @@ public function check()
         return $returnValue;
     }
 
-    /**
-     * If file is readable.
-     *
-     * @access public
-     * @author Aleh Hutnikau, <hutnikau@1pt.com>
-     * @param string $location File location
-     * @return boolean
-     */
-    public function isReadable($location = null)
+    private function hasLocationAccess($location = null, $rule = 'Readable')

```

Comment:
It's a private method, all rules are defined in this class as static string
What can I add to Doc? For me it's clear from function name